gpu/gpu.py.data/xgemm.c

```diff
@@ -0,0 +1,351 @@
+#ifdef USE_MPI
```

Comment:
Who is the author of this code?
Better to add some license agreement along with Author information.

Reply:
I think I got this from Pidad. Let me see how to add this, figuring out.